.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -37,7 +48,7 @@ body:
       label: Logs
       description: |
         If applicable (crash, error output in console), please provide your crash report or latest.log
-        To upload logs, use an external site to post the crash report, such as [Ubuntu Pastebin](https://paste.ubuntu.com/), [Pastebin](http://pastebin.com/), or [Github Gist](https://gist.github.com/)
+        To upload logs, use an external site to post the crash report, such as [**MCLogs (Recommended)**](https://mclo.gs/), [Ubuntu Pastebin](https://paste.ubuntu.com/), or [Pastebin](http://pastebin.com/)
```

Looks like paste.ubuntu.com requires registration now, and I could not find any information about its size limit. We should replace it with GitHub Gist instead, which still requires sign up but allows up to 100 MBs of data.
```suggestion
        To upload logs, use an external site to post the crash report, such as [**MCLogs (Recommended)**](https://mclo.gs/), [Github Gist](https://gist.github.com/), or [Pastebin](http://pastebin.com/)
```

---

How about we just remove paste.ubuntu.com, see no reason why we would need it tbh (same for gist). pastebin.com seems fine as an alternative, but can we use a https link instead?